Example/Custom Integration (ObjC)/BrowseExamplesViewController.m

```diff
@@ -36,24 +37,27 @@ - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
 }
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
-    return 4;
+    return 5;
 }
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
     UITableViewCell *cell = [UITableViewCell new];
     switch (indexPath.row) {
         case 0:
-            cell.textLabel.text = @"Card";
-            break;
+        cell.textLabel.text = @"Card";
+        break;
         case 1:
-            cell.textLabel.text = @"Card + 3DS";
-            break;
+        cell.textLabel.text = @"Card + 3DS";
+        break;
         case 2:
-            cell.textLabel.text = @"Apple Pay";
-            break;
+        cell.textLabel.text = @"Apple Pay";
+        break;
         case 3:
-            cell.textLabel.text = @"Sofort";
-            break;
+        cell.textLabel.text = @"Sofort";
+        break;
+        case 4:
+        cell.textLabel.text = @"Card + 3DS + PaymentIntent";
+        break;
```

Comment:
does `ctrl+i` give this indentation for obj-c now?

Reply:
It was, and I didn't like it but couldn't get it to do anything different. Now it looks like Xcode has changed its mind. So there must have been something I'd missed, maybe an error in a header or something.